testbed/tests/test_keys.py

```diff
@@ -14,7 +14,6 @@
         (Key.MOD_1 + "a", {"key": Key.A, "modifiers": {Key.MOD_1}}),
         (Key.MOD_2 + "a", {"key": Key.A, "modifiers": {Key.MOD_2}}),
         (Key.MOD_3 + "a", {"key": Key.A, "modifiers": {Key.MOD_3}}),
-        (Key.CAPSLOCK + "a", {"key": Key.A, "modifiers": {Key.CAPSLOCK}}),
```

Comment:
Although GTK and Cocoa treat caps lock as a modifier, WinForms does not. And it wouldn't make sense anyway in the context of command shortcuts -- nobody will ask for a shortcut of "Caps Lock + X".